config/dependencies.yaml

```diff
@@ -192,7 +192,7 @@ react-pattern:
     '@freesewing/i18n': *freesewing
 rehype-jargon:
   _:
-    'unist-util-visit': '^2.0.3'
+    'unist-util-visit': '^4.1.0'
```

Comment:
remark-jargon had at some point been bumped to `unist-util-visit@^4.1.0`, so I figured that they should both be on the same version, but tell me if I'm wrong?

Reply:
Not sure why it's different from remark-jargon. Or rather I can't remember.